charts/spire/README.md

```diff
@@ -7,6 +7,52 @@ A Helm chart for deploying the complete Spire stack including: spire-server, spi
 
 **Homepage:** <https://github.com/spiffe/helm-charts/tree/main/charts/spire>
 
+## Install notes
+
+To do a quick non production install suitable for quick testing in something like minikube:
+
+```shell
+kubectl create namespace spire-system
+helm install -n spire-system spire-crds --repo https://spiffe.github.io/helm-charts-hardened/
+helm install -n spire-system spire --repo https://spiffe.github.io/helm-charts-hardened/
+```
```

Comment:
Does the spire-crds need the namespace?
How about helm install --create-namespace?
Suggestion to use default namespace rather than overriding.

Reply:
True, its listed as non prod, so default is probably fine and easier to use. Will update.